learning/katas/go/Core Transforms/Branching/Branching/pkg/task/task.go

```diff
@@ -0,0 +1,58 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package task
+
+import (
+	"github.com/apache/beam/sdks/go/pkg/beam"
+	"strings"
+)
+
+func ApplyTransform(s beam.Scope, input beam.PCollection) (reversed, toUpper beam.PCollection) {
+	reversed = reverseString(s, input)
+	toUpper = toUpperString(s, input)
+	return
```

Comment:
Just a note, no action needed: You're using named values and naked returns correctly: in a very short function, to document the ordering of otherwise identical return values, but in larger functions, if can hurt readability. https://tour.golang.org/basics/7